modules/videoio/src/cap_ffmpeg.cpp

```diff
@@ -262,6 +265,29 @@ class CvCapture_FFMPEG_proxy CV_FINAL : public cv::IVideoCapture
 
 protected:
     CvCapture_FFMPEG* ffmpegCapture;
+private:
+    void rotateFrame(Mat &mat) const {
+        if(!ffmpegCapture) {
+            return;
+        }
+        int angle = ffmpegCapture->rotation_angle;
+        if(!ffmpegCapture->rotation_auto || (angle % 360) == 0) {
```

Comment:
OpenCV build for Windows used FFmpeg through dedicated wrapper DLL and internal fields are not visible. You have to use `getProperty` method to get values.